addons/mail/static/tests/tours/discuss_channel_call_public_tour.js

```diff
@@ -22,11 +22,10 @@ registry.category("web_tour.tours").add("discuss_channel_call_public_tour.js", {
         {
             content: "Check that the call has started",
             trigger: ".o-discuss-Call",
-            run() {
-                if (!odoo.__WOWL_DEBUG__.root.env.services["discuss.rtc"]?.selfSession) {
-                    console.error("The call should have started.");
-                }
-            },
+        },
+        {
+            content: "Check that the Disconnect button is visible",
```

Comment:
`Check that current user is in call ("disconnect" button visible)`

This step would feel less random by re-wording that way